# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/WebAuthnUtil.h
# Commit: eb8cc8d7badf
# Full Hash: eb8cc8d7badf8643681a35d38d88c751e08926fa
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-11 09:01:30
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r add154b24d33 -r eb8cc8d7badf dom/webauthn/WebAuthnUtil.h
--- a/dom/webauthn/WebAuthnUtil.h	Mon Feb 10 20:11:06 2025 +0000
+++ b/dom/webauthn/WebAuthnUtil.h	Mon Feb 10 20:11:06 2025 +0000
@@ -7,10 +7,6 @@
 #ifndef mozilla_dom_WebAuthnUtil_h
 #define mozilla_dom_WebAuthnUtil_h
 
-/*
- * Utility functions used by both WebAuthnManager and U2FTokenManager.
- */
-
 #include "mozilla/dom/WebAuthenticationBinding.h"
 #include "ipc/IPCMessageUtils.h"
 